ps/ps.go

```diff
@@ -26,7 +27,8 @@ func List(c *deis.Client, appID string, results int) ([]api.Pods, int, error) {
 	return procs, count, nil
 }
 
-// Scale an app's processes.
+// Scale an app's processes. The processes are specified in the target argument,
```

Comment:
this would be better written as `Scale increases or decreases an app's processes...`